exercises/joseph_ramsay/ex02_oo_basics/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+cmake_minimum_required ( VERSION 2.8.5 )
+
+###############################################################################
+# Setup
+###############################################################################
+# Name
+project ( Shapes )
+
+# Find custom cmake modules
+set ( CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../../../CMake")
+
+# Common setup
+include ( CommonSetup )
+
+###############################################################################
+# Executable
+###############################################################################
+
+set ( SRC_FILES
+  src/main.cpp
```

Comment:
The coding standard (for `c++`) in Mantid requires you to have capitalized and camel case file names (except for `main.cpp`).